waspc/src/Wasp/Util/Path.hs

```diff
@@ -0,0 +1,21 @@
+module Wasp.Util.Path
```

Comment:
You couldn't really have known this, but there is library for paths called Path. It was inspiratin for StrongPath, and is what StrongPath also uses internally for big part of its implementation (and then builds on top of it). It is relatively popular.

So we have FilePath, Path, StrongPath, sorted by how much they capture in types.

Since these are function operating on FilePath, not on Path, I would name module `Wasp.Util.FilePath`. Because we actually do use `Path` in some places in code, and even have a module for utility functions for it, called `Path` I believe or smth very similar.